store/storage/sqlite/db.go

```diff
@@ -19,6 +20,8 @@ const (
 	reservedStoreKey = "_RESERVED_"
 	keyLatestHeight  = "latest_height"
 
+	defaultBatchBufferSize = 100000
```

Comment:
can we add a comment describing why we this value was chosen?

Reply:
it is a just random value, we will tune this value later through the benchmarking